configuration/encountertypes/encounterTypes.csv

```diff
@@ -72,4 +72,6 @@ ${encounterType.NEWBORN_ADMISSION.uuid},,Newborn Admission,Newborn admission
 ${encounterType.NEWBORN_ASSESSMENT.uuid},,Newborn Assessment,Newborn assessment
 ${encounterType.NEWBORN_DISCHARGE.uuid},,Newborn Discharge,Newborn Discharge
 ${encounterType.INPATIENT_NOTE.uuid},,Inpatient Note,A short free-text note written by a clinician about an inpatient during their stay
-${encounterType.TRANSFER_REQUEST.uuid},,Transfer Request,A stand-alone encounter type for a transfer request not part of a larger note
\ No newline at end of file
+${encounterType.TRANSFER_REQUEST.uuid},,Transfer Request,A stand-alone encounter type for a transfer request not part of a larger note
+${encounterType.BED_ASSIGNMENT.uuid},,Bed Assignment,Assigning a new bed to a patient without admitting or transferring them
+${encounterType.CANCEL_ADT_REQUEST.uuid},,Cancel ADT Request,Cancellation of Admin Transfer or Discharge requests
```

Comment:
Sorry, if I didn't mention this, but let's just use the existing "CANCEL_ADMISSION" encounter type for this.